spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/service/SkipperStreamService.java

```diff
@@ -68,4 +70,6 @@
 	 * @return list of supported deployment platforms
 	 */
 	Collection<Deployer> platformList();
+
+	String convertPropertiesToSkipperYaml(StreamDefinition streamDefinition, Map<String, String> updateProperties);
```

Comment:
Should this be an interface method? If so, please add javadoc as to why we need it as a contract.

I believe this method can just be part of implementation as it sounds like a helper method.

Reply:
It is used by a test. I CANNOT use an implementation for dependency injection as was done by the original test or I get a proxy error. Either I add this method to the interface or we remove the test. I did not see how to solve the test error otherwise. @markpollack please advise.

Reply:
I will create the service 'by hand' to test the method and make it packag protected.